src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -61,9 +61,26 @@ def test_api_video_read_detail_anonymous(self):
             content, {"detail": "Authentication credentials were not provided."}
         )
 
+    def test_api_video_read_detail_student(self):
+        """Student users should not be allowed to read a video detail."""
+        video = VideoFactory()
+        jwt_token = AccessToken()
+        jwt_token.payload["video_id"] = str(video.id)
+        jwt_token.payload["roles"] = ["student"]
+        # Get the video linked to the JWT token
+        response = self.client.get(
+            "/api/videos/{!s}/".format(video.id),
+            HTTP_AUTHORIZATION="Bearer {!s}".format(jwt_token),
```

Comment:
Use f-strings!

Reply:
Ok I will look the documentation :smiley_cat: